Source/Dafny/Compilers/Compiler.cs

```diff
@@ -1273,6 +1275,15 @@ protected class NullClassWriter : IClassWriter {
       public ConcreteSyntaxTree/*?*/ CreateMethod(Method m, List<TypeArgumentInstantiation> typeArgs, bool createBody, bool forBodyInheritance, bool lookasideBody) {
         return createBody ? block : null;
       }
+
+      public ConcreteSyntaxTree CreateFreshMethod(Method m) {
```

Comment:
I think this is leftover from earlier revisions and can be deleted.

Reply:
Done!